### ISSUE TITLE
Conflict with laravel 5.7.9

### ISSUE THREAD

**Issue by AlessandroMinoccheri** [nWidart/Laravel-broadway#30]
I have conflict when I try to install this package. I have a laravel 5.7.9 application and when I try to launch composer command returns me this error:

````
Problem 1
    - Conclusion: remove laravel/framework v5.7.9
    - Conclusion: don't install laravel/framework v5.7.9
    - nwidart/laravel-broadway 2.0.0 requires illuminate/support 5.1.*|5.2.*|5.3.*|5.4.* -> satisfiable by illuminate/support[5.1.x-dev, 5.2.x-dev, 5.3.x-dev, 5.4.x-dev, v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.30, v5.1.31, v5.1.41, v5.1.6, v5.1.8, v5.2.0, v5.2.19, v5.2.21, v5.2.24, v5.2.25, v5.2.26, v5.2.27, v5.2.28, v5.2.31, v5.2.32, v5.2.37, v5.2.43, v5.2.45, v5.2.6, v5.2.7, v5.3.0, v5.3.16, v5.3.23, v5.3.4, v5.4.0, v5.4.13, v5.4.17, v5.4.19, v5.4.27, v5.4.36, v5.4.9].
    - nwidart/laravel-broadway 2.0.1 requires illuminate/support 5.1.*|5.2.*|5.3.*|5.4.*|5.5.* -> satisfiable by illuminate/support[5.1.x-dev, 5.2.x-dev, 5.3.x-dev, 5.4.x-dev, 5.5.x-dev, v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.30, v5.1.31, v5.1.41, v5.1.6, v5.1.8, v5.2.0, v5.2.19, v5.2.21, v5.2.24, v5.2.25, v5.2.26, v5.2.27, v5.2.28, v5.2.31, v5.2.32, v5.2.37, v5.2.43, v5.2.45, v5.2.6, v5.2.7, v5.3.0, v5.3.16, v5.3.23, v5.3.4, v5.4.0, v5.4.13, v5.4.17, v5.4.19, v5.4.27, v5.4.36, v5.4.9, v5.5.0, v5.5.16, v5.5.17, v5.5.2, v5.5.28, v5.5.33, v5.5.34, v5.5.35, v5.5.36, v5.5.37, v5.5.39, v5.5.40, v5.5.41, v5.5.43, v5.5.44].
    - illuminate/support 5.2.x-dev conflicts with laravel/framework[v5.7.9].
    - illuminate/support 5.3.x-dev conflicts with laravel/framework[v5.7.9].
    - illuminate/support 5.4.x-dev conflicts with laravel/framework[v5.7.9].
    - illuminate/support 5.5.x-dev conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.2.43 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.2.45 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.3.0 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.3.16 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.3.23 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.3.4 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.4.0 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.4.13 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.4.17 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.4.19 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.4.27 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.4.36 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.4.9 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.0 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.16 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.17 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.2 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.28 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.35 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.36 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.37 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.39 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.40 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.41 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.43 conflicts with laravel/framework[v5.7.9].
    - illuminate/support v5.5.44 conflicts with laravel/framework[v5.7.9].
    - don't install illuminate/support 5.1.x-dev|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.1|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.13|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.16|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.2|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.20|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.22|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.25|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.28|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.30|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.31|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.41|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.6|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.1.8|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.0|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.19|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.21|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.24|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.25|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.26|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.27|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.28|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.31|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.32|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.37|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.6|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.2.7|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.5.33|don't install laravel/framework v5.7.9
    - don't install illuminate/support v5.5.34|don't install laravel/framework v5.7.9
    - Installation request for laravel/framework v5.7.9 -> satisfiable by laravel/framework[v5.7.9].
    - Installation request for nwidart/laravel-broadway ^2.0 -> satisfiable by nwidart/laravel-broadway[2.0.0, 2.0.1].

```

**Comment by Lavoaster** [comment]
Can you try again? I just released v2.0.2 which updates the version constraints. Let me know if you run into any issues and I'll resolve them.

**Comment by AlessandroMinoccheri** [comment]
Now I have tried and with version 2.0.2 I don't get any conflict, so now it's time to write some code
